packages/aserto/test/test_directory.py

```diff
@@ -104,27 +104,26 @@ def test_object_not_found(directory):
 
 
 def test_object_invalid_arg(directory):
-    with pytest.raises(TypeError):
-        directory.client.get_object(key=directory.obj_1.key)
+    with pytest.raises(grpc.RpcError, match="object identifier invalid argument") as err:
+        directory.client.get_object(key=directory.obj_1.key, type="")
 
 
+@pytest.mark.skip(reason="topaz directory doesn't filter on type")
 def test_get_objects_by_type(directory):
     objs = directory.client.get_objects(
         object_type=directory.obj_1.type, page=PaginationRequest(size=10)
     ).results
-    print(objs)
     assert directory.obj_1 in objs
     assert directory.obj_1.type == directory.obj_3.type
     assert directory.obj_1.type != directory.obj_2.type
     assert directory.obj_3 in objs
-    # assert directory.obj_2 not in objs
+    assert directory.obj_2 not in objs
 
 
 def test_get_objects(directory):
     objs = directory.client.get_objects(page=PaginationRequest(size=10)).results
     assert directory.obj_1 in objs
-    assert directory.obj_2 in objs
-    assert directory.obj_3 in objs
+    assert len(objs) == 3
```

Comment:
I think you do want to keep the original assertions in addition to the length check in this test case because you're validating that all three expected objects are returned and only those objects.